checking Feature branch for stickler activation.

jamestucker1988 opened the pull request:
xyz

jamestucker1988 commented:
-The given project is a curriculum-based project. All the objectives outlined in the instructions are achieved- namely, using grid,flex-box,float to clone the newyork times webpage. 
-The page consists of a navbar,body with article tags and a footer section .

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,335 @@
+
+        </div>
+        <div class="underline"></div>
+    </div>
```
There is an element that is missing and is the bottom section of the page (that sticks to the bottom and says expand).

I am attaching a screenshot from the original page.

![image](https://user-images.githubusercontent.com/57414789/81358085-197a5680-909b-11ea-92ff-21d906c3a5b2.png)

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,335 @@
+        </div>
+        <div class="underline"></div>
+    </div>
 </body></html>
```
There are some warnings in the HTML validator. Kindly fix them.

> 
![image](https://user-images.githubusercontent.com/57414789/81358193-6eb66800-909b-11ea-91cd-466002db0627.png)

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,335 @@
-            <li><span class="fas fa-bars align1"></span></li>
-            <li><span class="fa fa-search align1"></span></li>
-            <li>space&amp;cosmos</li>
-            <li><img src="newyork.svg"> </li>
+        <ul class="nav-ul">
+            <li class="nav-left"><span class="fas fa-bars
+                "></span></li>
+            <li class="nav-left"><span class="fa fa-search
+                "></span></li>
+            <li class="nav-left">space&amp;cosmos</li>
+        </ul>
+        <img src="image/newyork.svg" alt="unable to load">
+        <ul class="nav-ul">
+            <li><button>Subscribe Now</button></li>
             <li><button>log in</button></li>
         </ul>
     </nav>
```
The navbar doesn't look like the one on the original webpage.

I will attach a screenshot. 

> 
![image](https://user-images.githubusercontent.com/57414789/81358363-e97f8300-909b-11ea-9c91-3e342f67547f.png)

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,335 @@
+        <div class="date">
+            <p>March 17, 2014</p>
+            <ul>
+                <li><i class="fab fa-facebook"></i></li>
+                <li><i class="fab fa-twitter-square"></i></li>
+                <li><i class="fas fa-envelope"></i></li>
+                <li><i class="fas fa-share"></i></li>
+                <li> <i class="medium material-icons">bookmark</i></li>
```
These icons should be links. You can wrap the icon inside an anchor tag (```<a href="#"><i></i></a>```).

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,335 @@
+    <div class="grid-container">
+        <p><strong>The Theory of Inflation</strong><br>
+            Astronomers have found evidence to support the theory of inflation, which explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.</p>
+        <div class="grid-background ">
+            <p><strong>THE UNIVERSE</strong><br> is just under 14 billion years old. From our position in the Milky Way galaxy, we can observe a sphere that is now about 92 billion light-years across. But there's a mystery. Wherever we look, the universe has an even temperature.</p>
+            <p> <strong>NOT ENOUGH TIME</strong>The universe is not old enough for light to have traveled the vast distance from one side of the universe to the other, and there has not been enough time for scattered patches of hot and cold to mix into an even temperature.</p>
+            <p><strong>DISTANT COFFEE</strong><br> At a smaller scale, imagine using a telescope to look a mile in one direction. You see a coffee cup, and from the amount of steam, you can estimate its temperature and how much it has cooled.</p>
+            <p><strong>COFFEE EVERYWHERE</strong><br>Now turn around and look a mile in the other direction. You see a similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a similar cup in every direction, you might want to look for another explanationbottom-c.</p>
+            <p><strong>STILL NOT ENOUGH TIME </strong><br>There has not been enough time to carry coffee cups from place to place before they get cold. But if all the coffee cups were somehow filled from a single coffee pot, all at the same time, that might explain their even temperature.</p>
+            <p><strong>INFLATION</strong><br>solves this problem. The theory proposes that, less than a trillionth of a second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in the violently expanding energy field eventually grew into the large-scale structures of the universe.</p>
+            <p><strong>FLUCTUATION</strong><br> Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.</p>
+            <p><strong>EXPANSION</strong><br>Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.</p>
+        </div>
```
This text sometimes is over the images. Please, fix that.

> 
![image](https://user-images.githubusercontent.com/57414789/81358571-8c380180-909c-11ea-851e-b4b9d2362597.png)

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,335 @@
+                <button>READ 615 COMMENTS</button>
+
+                <ul>
+                    <li><i class="fab fa-facebook "></i></li>
+                    <li><i class="fab fa-twitter-square "></i></li>
+                    <li><i class="fas fa-envelope "></i></li>
+                    <li><i class="fas fa-share "></i></li>
```
These icons should be links as well.

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,335 @@
+            <aside>
+
+
+                <div>Finally There’s Time to Read (or at Least Organize Your Books)</div>
+                <div>A Father’s Love, Whether He Says Too Much or Not Enough</div>
+                <div>Polls Had Trump Stewing, and Lashing Out at His Own Campaign</div>
+                <div>De Blasio Breaks Up Rabbi’s Funeral and Lashes Out Over Virus Distancing</div>
+                <div>Why Zoom Is Terrible</div>
+                <div>This Florida Student Was Accepted at All 8 Ivy League Schools</div>
+                <div>Opinion: At Least 89 Vaccines Are Being Developed. It May Not Matter.</div>
+                <div>What Phone Calls Have Given Me That Video Chat Can’t</div>
+                <div>Top E.R. Doctor Who Treated Virus Patients Dies by Suicide</div>
+                <div>Opinion: They Didn’t Drink the Bleach, but They’re Still Drinking the Kool-Aid</div>
```
This text looks different from the original. Kindly fix that.

Original
> 
![image](https://user-images.githubusercontent.com/57414789/81358715-e33dd680-909c-11ea-9932-48758da3483f.png)

Your page
> 
![image](https://user-images.githubusercontent.com/57414789/81358748-f94b9700-909c-11ea-87ad-b00d84713a58.png)

lucasmazo32 commented on style.css:
```diff
@@ -0,0 +1,379 @@
+.main-content {
+  position: relative;
+  padding-top: 8rem;
+
+  /*  padding-left: 5rem; */
```
Please erase this commented code.

jamestucker1988 commented:
I introduced the changes recommended by TSE,namely
1-remove validation warnings.
2-modified nav element.
3-added anchor tags to the social icon.
4- fixed background image in grid-container.
5-changing texts "Due to dynamic rendering of webpage the contents change everytime  u access or few hours I have when I first accessed it."
6-erased the commented code.

jamestucker1988 commented:
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
1-remove validation warnings. And indentation errors.
2-modified nav element.
3-Added anchor tags to the social icon.
4- fixed background image in the grid-container.
5-erased the commented code.

**HERE is a preview link:<a  href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">here</a>**

jamestucker1988 commented:
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
-I have changed navigation "Humberger" icon to be a button.
 -changed "SPACE & COSMOS" in the navigation bar should to be a link.
 -removed unnecessary white space above your navigation bar.
 -removed  horizontal scroll bar.
- "More in Space and Astronomy" section has links to other articles. 
- "Most Popular" section has links to more popular articles. 
 -The bottom section with the logo and other navigation links have been made links.
 -the clone has a footer that sticks to the bottom part of the screen. 

**HERE is a live preview <a href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">click here</a>**

sumancrest0001 commented:
**Status: changes required
Hello @jamestucker1988** 
You have done a good job to fix the issues raised by the previous TSEs but we still need to make a couple of changes to get :+1:  for this project. 
**[NOTE] All the requirements are mandatory.**

sumancrest0001 commented:
Let's fix the navbar issue. change the space & cosmos font to match the original one, add a background color to buttons, and other sections are overlapping with the navbar when we scroll.
Have a look :point_down: 
![Screenshot from 2020-05-13 08-37-28](https://user-images.githubusercontent.com/37747619/81766738-6a080e80-94f6-11ea-8985-bc65637d57c2.png)

sumancrest0001 commented:
The main section should be centered and width should be similar to the original one. currently, your main section is narrower.

The More in Space and Astronomy section should have 3 article column but it's a two-column section in your project. Please consider the following image to get clear.
![Screenshot from 2020-05-13 08-42-00](https://user-images.githubusercontent.com/37747619/81767005-16e28b80-94f7-11ea-843c-5bf8f3a8f24e.png)

sumancrest0001 commented:
Let's fix the alignment of items in the following sections of the footer. They should be aligned at left.
![Screenshot from 2020-05-13 08-39-59](https://user-images.githubusercontent.com/37747619/81767084-4396a300-94f7-11ea-95be-7278687d15c4.png)

But currently :point_down: 
![Screenshot from 2020-05-13 08-54-26](https://user-images.githubusercontent.com/37747619/81767135-645ef880-94f7-11ea-99fc-4139a937e4fb.png)

sumancrest0001 commented:
we have to fix  the overlapping problem with the sticky footer section also. 
[TIPS] Add background color to white.

jamestucker1988 commented:
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
1-fixed overlapping problem with sticky footer.
2-alignment items in footer section.
3-more in space and astronomy now has three columns.
4-changed fontsize,color in navbar.

**HERE is a preview link:<a  href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">here</a>**

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,340 @@
 
+            <a href="#">space &amp; cosmos</a>
 
+        </div>
+        <img src="image/newyork.svg" alt="unable to load">
+        <div class="nav-ul">
```
- [ ] Kindly wrap the logo inside an anchor tag `<a></a>` as the original page.

lucasmazo32 commented on index.html:
```diff
@@ -1,26 +1,340 @@
+                    <li><a href="#">Tools &amp; Services</a></li>
+                    <li><a href="#"> N.Y.C. Events Guide</a></li>
+                    <li><a href="#">Multimedia</a></li>
+                    <li><a href="#"> Photography</a></li>
+                    <li><a href="#">Video</a></li>
+                    <li><a href="#">Newsletters</a> </li>
+                    <li><a href="#">Times Journeys</a> </li>
+                    <li><a href="#">NYT Store</a> </li>
+                    <li><a href="#">Manage My Account</a> </li>
+                </ul>
+                <ul class="ul">
+                    <li><strong><a href="#" class="strong">SUBSCRIBE</a></strong></li>
+                    <li><strong><a href="#" class="strong">Home Delivery</a></strong></li>
+                    <li><strong><a href="#" class="strong">Digital Subscriptions</a></strong> </li>
+                    <li>
+                        <strong><a href="#" class="strong">Crossword</a></strong></li>
+                    <li><strong><a href="#" class="strong">Cooking</a></strong> </li>
+                    <li class="li-small"><a href="#">Email Newsletters</a></li>
+                    <li class="li-small"><a href="#">Corporate Subscriptions</a></li>
+                    <li class="li-small" id="li-small1"><a href="#">Education Rate</a></li>
+                    <li class="li-small"><a href="#">Replica Edition</a> </li>
+                    <li class="li-small"><a href="#">Mobile Applications</a></li>
+                    <li class="li-small"><a href="#">中文网</a></li>
+                    <li class="li-small"><a href="#">Español</a></li>
+                </ul>
+            </div>
+            <ul class="ul">
+                <li><a href="#">© 2020 The New York Times Company</a></li>
+                <li><a href="#">NYTCo</a></li>
+                <li><a href="#">Contact</a></li>
+                <li><a href="#">Work with us</a></li>
+                <li><a href="#">Advertise</a></li>
+                <li><a href="#">T Brand Studio</a></li>
+                <li><a href="#">Your Ad Choices</a></li>
+                <li><a href="#">Privacy</a></li>
+                <li><a href="#">Terms of Service</a></li>
+                <li><a href="#">Site Map</a></li>
+                <li><a href="#">Help</a></li>
+                <li><a href="#">Subscriptions</a></li>
+                <li><a href="#">SaleSite</a></li>
```
- [ ] Please check this section. Some of the words are overlaping.

> ![image](https://user-images.githubusercontent.com/57414789/81999879-2e676480-961c-11ea-89c2-0108dba40820.png)

jamestucker1988 commented:
The given project is a curriculum-based project where I have cloned the new york times webpage. I have divided my webpage into the nav,main-content, footer—used modern web techniques like grid, flex, and float to style the webpage. The use of aside tag has been specified in the instructions. As such, I have used it in my project. As such, I have used it in my project.
-I have also made the following changes in my submission namely,
1-wrapped the logo inside an anchor tag <a></a> as the original page.
2-removed word overlapping in the indicated section.

**HERE is a preview link:<a  href="https://raw.githack.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/feature/index.html">here</a>**

rammazzoti2000 commented:
## Project Approved :rocket: :tada: 
Hello team great job, your project is approved, feel free to merge whenever you are ready.
###
Cheers and Happy coding! :clap: 

##### Code review by Alexandru Bangau

(Feel free to contact me on slack by the handle of alexandru if you have any additional question)</em>
### :heavy_check_mark: Status: Approved

rammazzoti2000 commented:
Hello @jamestucker1988, you shouldn't close the pull request but to merge the branch, reopen this pull request and than merge, after which submit your form for the completion of the project
For more clarifications you can find me on slack under the name of alexandru

jamestucker1988 commented:
I am reopening the pull request.

jamestucker1988 commented:
I closed and merged it with the feature branch.
**here is a new pull request I have create<a href="https://github.com/jamestucker1988/PROJECT-POSITIONING-AND-FLOATING-ELEMENTS/pull/4">click here</a>**
